src/noisepy/seis/io/scedc_s3store.py

```diff
@@ -175,3 +175,8 @@ def _parse_channel(filename: str) -> Channel:
             # lat/lon/elev will be populated later
             Station(network, station, location=location),
         )
+
+
+timespan1 = DateTimeRange(
```

Comment:
Similar to the above, let's not put test code in the package. I think the both `channelfilterstore.py` and `test_channelfilterstore.py` should move to `noisepy-seis-io` since they are both IO related.